ister.py

```diff
@@ -423,6 +423,8 @@ def copy_os(args, template, target_dir):
     swupd_command += " --contenturl={0}".format(args.contenturl)
     swupd_command += " --versionurl={0}".format(args.versionurl)
     swupd_command += " --format={0}".format(args.format)
+    if args.fast_install:
+        args.statedir = "{0}/swupd".format(target_dir)
```

Comment:
This will give you `<target_dir>/swupd` instead of `<target_dir>/var/lib/swupd` which is what I assume you want.

Reply:
Ah, you remove it later on after install is complete, so it doesn't really matter.

Reply:
But in case we decide there is going to be a swupd directory in / someday I'd probably use tempfile.mkdtemp with a prefix using the date or something to ensure this won't cause problems.

Reply:
Actually, to simply life in testing, just include the date in the directory name.